README.md

```diff
@@ -123,7 +123,7 @@ Options:
   -A <name>                     Export the arguments as a Bash 4.x associative
                                 array called <name>.
   -G <prefix>                   Don't use associative array but output
-                                Bash 3.x compatible GLOBAL variables assignment:
```

Comment:
The `Options` section of the README is extracted from compiled `docopts --help` as suggested my the "magic line above" :

https://github.com/docopt/docopts/blame/master/README.md#L99 

change it in [docopts.go](https://github.com/docopt/docopts/blob/master/docopts.go#L59) 

then

```
make README.md
```

Not documented yet it's experimental.

Reply:
I've reverted this change so it matches the Go code - not important.